### ISSUE TITLE
[Feature Request]I18n Support for Tooltips and GUIs' Tips

### ISSUE THREAD

**Issue by SihenZhang** [Bartz24/SkyResources#100]
There are some hardcoded tooltips and GUIs' Tips for some machines just like below.

https://github.com/Bartz24/SkyResources/blob/70fb53a2600e14b2a2ffbc4e2af132ec64fffbe1/src/main/java/com/bartz24/skyresources/base/MachineVariants.java#L134-L143

https://github.com/Bartz24/SkyResources/blob/7f53d1ffdfbfd80b9ab10d32343a679be85dcf2f/src/main/java/com/bartz24/skyresources/alchemy/gui/GuiLifeInfuser.java#L106-L109

However, when using Chinese, the tooltips will become just like this.

![](https://i.imgur.com/yQtR4NE.png)

Though these will not be too hard to understand for some players who know a little English, it is awful that Chinese and English are being put together.

I know that improving the tooltips may cost lots of time, but I am looking forward to have it done. And if more i18m were added, I'd like to translate it.

**Comment by Bartz24** [comment]
Yeah, this is on my to do list, but I'm focusing on adding features (and changing some of stat display of machines) before I clean up translation stuff.
